CTP/shell/src/com/navercorp/cubridqa/shell/main/Main.java

```diff
@@ -44,6 +44,10 @@ public static void exec(String configFilename) throws Exception {
 		// system.setProperty("sun.rmi.transport.tcp.responseTimeout", "0");
 
 		Context context = new Context(configFilename);
+		if(CommonUtils.isEmpty(context.getTestCaseRoot())){
+			throw new Exception(
+					"[ERROR]: scenario is null!");
```

Comment:
Refine message:
as is:
[ERROR]: scenario is null!
to be:
[ERROR] The parameter 'scenario' must be set in configuration file.